config/environments/development.rb

```diff
@@ -52,10 +52,26 @@
   # Suppress logger output for asset requests.
   config.assets.quiet = true
 
+  # Force https
+  config.force_ssl = true
```

Comment:
Does this also enable HSTS on local machines? If so, it makes it hard to run other things on `localhost` that don't use HTTPS.

Reply:
Looks like we can disable the HSTS part, by setting:

`config.ssl_options = { hsts: false }`

https://api.rubyonrails.org/v5.2.3/classes/ActionDispatch/SSL.html

@erbridge do you fancy exlpaining why HSTS could be an issue? I've never come accross it before?

Reply:
HSTS basically tells your browser to always redirect that URL to HTTPS. This is fine in real environments, since once you add SSL, you rarely remove it, but for `localhost`, it means that your browser will try to upgrade all `http://localhost` requests to `https://localhost`, which might not be there. And it can be a real pain to get your browser to forget that once it's happened.

Reply:
@erbridge Good catch, I've just checked and local https will still work without this config setting so I've removed it

Reply:
We should be forcing SSL in production, though. How about something like:
```
config.force_ssl = true
config.ssl_options = { hsts: Rails.env.production? }
```